src/assets/styles/_foundation.scss

```diff
@@ -246,7 +246,6 @@ $mobile-margin: 16px;
 
 .strikethrough {
   position: relative;
-  display: inline-block;
```

Comment:
Note to reviewer, updated so this class works w/ new font to look more like current production site

`'PPNeueBit-Bold'` with `display: inline-block;`
![Screenshot 2024-08-05 at 3 57 05 PM](https://github.com/user-attachments/assets/d8a9dff6-a1a9-4e0c-8e87-f41a039d36bf)

`'PPNeueBit-Bold'` with default `display` property
![Screenshot 2024-08-05 at 3 57 09 PM](https://github.com/user-attachments/assets/bb16695d-5e9c-4f8c-b0a4-9a99a59c13b7)